### PR TITLE
#162725213 Setup testing environment

### PR DESCRIPTION
**Pull request by malfahad**
#### What does this PR do?
Sets up the testing environment with Jest and Enzyme.

#### Description of Task to be completed?
-Install all dependencies
-write tests for existing code

#### How should this be manually tested?
 - Copy this branch to your local machine with `git fetch origin ch-setup-tests-162725213 && git checkout ch-setup-tests-162725213`
 - Install dependencies with `yarn install`
 - Run tests with `yarn test`
 - Run coverage with `yarn test:coverage`

#### What are the relevant pivotal tracker stories?
[#162725213](https://www.pivotaltracker.com/story/show/162725213)

### CODE REVIEW

**Review by MuhweziDeo — DISMISSED** [review]
LGTM

**Comment by kdaglas** [comment]
@malfahad so now that you fixed your hound issue, please click resolve on all those hound comments so that you reduce our scrolling, am just saying, but i have tested and all is good, the test env is functional.

**Review by calvinpete — DISMISSED** [review]
LGTM

**Review by kdaglas — DISMISSED** [review]

**Review by mariamiah — DISMISSED** [review]
LGTM

**Review by d-rita — DISMISSED** [review]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ node_modules
 package-lock.json
 dist
 yarn.lock
+coverage
```
Comment (RrNn) [inline .gitignore]: We need a blank new line here.

**Review by RrNn — DISMISSED** [review]

**Comment by mariamiah** [comment]
@malfahad kindly rebase and sort the conflicting files

**Review by mariamiah — APPROVED** [review]
LGTM

**Review by calvinpete — APPROVED** [review]
LGTM

**Review by kdaglas — APPROVED** [review]
i have tested and the tests are doing ok

**Review by RrNn — APPROVED** [review]
LGTM

**Review by d-rita — APPROVED** [review]

**Pull request merged**